src/AppFrame.tsx

```diff
@@ -31,9 +44,22 @@ export function AppFrame() {
             items={[
               {
                 url: "/",
-                label: "Home",
+                label: t("home"),
                 icon: HomeMajor,
               },
+              {
+                label: languageToggle(),
+                onClick: () => {
+                  if (
+                    i18n.language !== undefined &&
+                    i18n.language.substring(0, 2) === "fr"
```

Comment:
Minor, could be shortened to `i18n.language?.substring(0, 2) === "fr"`

Reply:
eslint doesn't complain :)